angular/core/css/1_base.scss

```diff
@@ -28,3 +28,7 @@ a {
   color: $link-color;
   text-decoration: none;
 }
+
+*:focus {
+    outline: none;
```

Comment:
I'm seeing some a11y complaints about this one:

http://www.outlinenone.com/
http://a11yproject.com/posts/never-remove-css-outlines/